packages/react-core/src/evm/providers/thirdweb-sdk-provider.tsx

```diff
@@ -179,11 +179,17 @@ export const ThirdwebSDKProvider = <TChains extends Chain[]>({
   queryClient,
   supportedChains,
   activeChain,
-  thirdwebApiKey = DEFAULT_API_KEY,
+  thirdwebApiKey,
   alchemyApiKey,
   infuraApiKey,
   ...restProps
 }: React.PropsWithChildren<ThirdwebSDKProviderProps<TChains>>) => {
+  if (!thirdwebApiKey) {
+    console.warn(
+      "No API key provided. You will have limited access to thirdweb's services for storage, RPC, and account abstraction. You can get an API key from https://thirdweb.com/dashboard/",
```

Comment:
same here - we don't want to print it twice betwee ThirdwebProvider / ThirdwebSDKProvider

In fact i think we can just print it here, but we need to do a useRef or something to not print on every render

Reply:
Notice that this will only happen if the user users sdk directly, for react/rn, we catch this in those packages and pass a default so this will not trigger.

Will look at the re-rendering

Reply:
fixed